responsibleai/responsibleai/_tools/shared/attribute_serialization.py

```diff
@@ -113,8 +113,13 @@ def _load_attribute(
     file_format: str,
 ) -> Any:
     if file_format == SerializationFormats.PICKLE:
-        with open(path, 'rb') as f:
-            return pickle.load(f)
+        val = None
+        try:
+            with open(path, 'rb') as f:
+                val = pickle.load(f)
+        except Exception:
+            pass
```

Comment:
Should we at least write out the exception as a warning?

Reply:
Done